build/lib/stylelint/vscode-known-variables.json

```diff
@@ -721,6 +721,8 @@
 		"--vscode-testing-uncoveredBorder",
 		"--vscode-testing-uncoveredBranchBackground",
 		"--vscode-testing-uncoveredGutterBackground",
+		"--vscode-testing-uncoveredGutterBackground",
+		"--vscode-testing-coverage-lineHeight",
```

Comment:
This seeems accidental

Reply:
<img width="944" alt="image" src="https://github.com/microsoft/vscode/assets/38986298/0176454e-d167-42e4-8e34-6dfd9f09b482">
When I try to remove the code in `src/vs/workbench/contrib/testing/browser/media/testing.css` the `husky` lint don't let me proceed the commit process. So I add this line of code to this file.